SchemaDatabase/SGr/Product/Product.xsd

```diff
@@ -27,6 +27,7 @@ https://github.com/SmartgridReady/SGrSpecifications  -->
     <sequence>
       <element name="name" type="string" />
       <element name="dataType" type="sgr:DataTypeProduct" />
+      <element name="defaultValue" type="string" />
```

Comment:
I'ld make the defaultValue optional with `minOccurs="0"`. Then you don't need to define it everywhere.